README.md

```diff
@@ -22,32 +28,37 @@ This is an out-of-the-box implementation of Wordpress.  It's an example of how c
     * Template: `cf create-service SERVICE PLAN SERVICE_INSTANCE`
     * Example: `cf create-service aws-rds shared-mysql mysql-service`
 
-3. Create a service instance of S3 storage. NB: A demo account will not be permitted to use the S3 service, so you can skip this for a basic demo.
+3. Create a service instance of S3 storage.
+
+cloud.gov does not have persistent local storage so you'll need to rely on S3 for storing any files uploaded to WordPress. Sandbox accounts cannot create S3 storage services, consider upgrading to a prototyping package if you need to do this.
```

Comment:
Split the second sentence into two at the comma.

Reply:
✅